lib/publiccloud/instance.pm

```diff
@@ -287,6 +288,7 @@ sub softreboot
     sleep 60;    # wait for the +1 in the previous command
     my $start_time = time();
 
+    sleep($args{delay});
```

Comment:
This is difficult. This routine is also used for measuring of boottime (see the line before), so adding an additional delay here might interfere with the boottime test. We could update line 288 though and state, that if delay is set, this subroutine might not return the accurate boottime.